definitions/procedures/pulp/remove.rb

```diff
@@ -73,6 +75,53 @@ def remove_mongo
       end
     end
 
+    # rubocop:disable Metrics/BlockLength, Metrics/MethodLength
+    def drop_migration_tables
+      with_spinner('Dropping migration tables') do
+        db_shell_input = <<-DBCOMMAND
+        BEGIN;
+        DROP TABLE IF EXISTS pulp_2to3_migration_migrationplan,
+          pulp_2to3_migration_pulp2blob,
+          pulp_2to3_migration_pulp2content,
+          pulp_2to3_migration_pulp2debcomponent,
+          pulp_2to3_migration_pulp2debcomponentpackage,
+          pulp_2to3_migration_pulp2debpackage,
+          pulp_2to3_migration_pulp2debrelease,
+          pulp_2to3_migration_pulp2debreleasearchitecture,
+          pulp_2to3_migration_pulp2distribution,
+          pulp_2to3_migration_pulp2distributor,
+          pulp_2to3_migration_pulp2distributor_pulp2_repos,
+          pulp_2to3_migration_pulp2distributor_pulp2_repos_id_seq,
+          pulp_2to3_migration_pulp2erratum,
+          pulp_2to3_migration_pulp2importer,
+          pulp_2to3_migration_pulp2iso,
+          pulp_2to3_migration_pulp2lazycatalog,
+          pulp_2to3_migration_pulp2manifest,
+          pulp_2to3_migration_pulp2manifestlist,
+          pulp_2to3_migration_pulp2modulemd,
+          pulp_2to3_migration_pulp2modulemddefaults,
+          pulp_2to3_migration_pulp2packagecategory,
+          pulp_2to3_migration_pulp2packageenvironment,
+          pulp_2to3_migration_pulp2packagegroup,
+          pulp_2to3_migration_pulp2packagelangpacks,
+          pulp_2to3_migration_pulp2repocontent,
+          pulp_2to3_migration_pulp2repository,
+          pulp_2to3_migration_pulp2rpm,
+          pulp_2to3_migration_pulp2srpm,
+          pulp_2to3_migration_pulp2tag,
+          pulp_2to3_migration_pulp2yumrepometadatafile,
+          pulp_2to3_migration_reposetup CASCADE;
+        END;
+        DBCOMMAND
+
+        db_shell_cmd = 'PULP_SETTINGS=/etc/pulp/settings.py pulpcore-manager dbshell'
```

Comment:
Can we add method(s) in PulpcoreDatabase to execute queries for pulpcoredb?

Reply:
@upadhyeammit Great idea!